participants/Pufferfish Cloud Storage Drive.md

```diff
@@ -0,0 +1,53 @@
+# <Pufferfish Cloud Storage Drive>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+Project Name: Pufferfish Cloud Storage Drive, Our slack Handle: U017LEUM8HF
+Username: 2966696199
+
+## (1) Brief Project Description
+
+Puffer Cloud Storage Drive is a network cloud storage Drive, It's public open to any user from all over the world users. Who can sign up and get 8GB free storage space. Puffer Cloud storage different with traditional cloud storage provider like Dropbox ,Baidu Drive, it's use blockchain technology to make payment and use it in id identifications. It's physical storage based on Skynet storage. We glad to add Filecoin physical storage infrastructure to our user.
+We have three team work together and use different technique to access Filecoin storage pool. Now we'd like to use lotus and powergate attend SR2 competition. 
+
+## (2) Link to Project UI
+
+Who can use this web URL to access all features of Puffer Drive.
+http://www.pufferdrive.com
+
+## (3) What does your application/UI do?
+Puffer Drive front-end developed by Typescript + React + Ant Design 4.0 + OOP, User can use it's UI to upload/download/share/manage files. It's friendly.

```

Comment:
Yes it is! I just created an account, and used it, works pretty well!

Reply:
Thank you.